app/controllers/AMIable.scala

```diff
@@ -46,7 +46,10 @@ class AMIable @Inject()(override val amiableConfigProvider: AmiableConfigProvide
         amiWithUpgrade = Recommendations.amiWithUpgrade(agents.allAmis)(ami)
         instances <- Prism.imageUsage(ami)
         launchConfigs <- Prism.launchConfigUsage(ami)
-      } yield Ok(views.html.ami(amiWithUpgrade, instances, launchConfigs))
+      } yield Ok(views.html.ami(
+        amiWithUpgrade,
+        PrismLogic.sortInstancesByStack(instances),
+        PrismLogic.sortLCsByOwner(launchConfigs)))
```

Comment:
Yes that is a very good option! I did it like that because I felt it was consisted with `Prismlogic.sortSSAAmisByAge` but I am happy to refactor!